syft/codes.py

```diff
@@ -1,24 +1,24 @@
-class PLAN_CMDS(object):
+class PLAN_CMDS(object):  # noqa: N801
```

Comment:
I think it would be okay to adopt the CapWords formatting suggested by N801 here.